server/src/main/java/org/opensearch/search/pipeline/Pipeline.java

```diff
@@ -132,7 +138,14 @@ SearchRequest transformRequest(SearchRequest request) throws SearchPipelineProce
                         request = processor.processRequest(request);
                     } catch (Exception e) {
                         onRequestProcessorFailed(processor);
-                        throw e;
+                        if (processor.getIgnoreFailure()) {
+                            processorFailureCount = processorFailureCount + 1;
+                            logger.info("failed to process request process: " + processor.getType());
+                            logger.error("An error occurred in the processor", e);
+                            logger.info("the number of failing pipeline increments to " + processorFailureCount);
+                        } else {
+                            throw e;
+                        }
```

Comment:
We should log the exception (but should probably log as `warn`, not `error`, since the pipeline configurator has specified that exceptions can be ignored), but we don't need to count anything here or log anything at `info` level.

The counting is already handled by `PipelineWithMetrics` in the `onRequestProcessorFailed` call.